core/src/test/scala/unit/kafka/log/LogTestUtils.scala

```diff
@@ -247,7 +246,7 @@ object LogTestUtils {
   }
 
   def listProducerSnapshotOffsets(logDir: File): Seq[Long] =
-    ProducerStateManager.listSnapshotFiles(logDir).map(_.offset).sorted
+    ProducerStateManager.listSnapshotFiles(logDir).asScala.map(_.offset).sorted.toSeq
```

Comment:
Is the `toSeq` at the end required? I'd have thought `sorted` returns a `Seq`.

Reply:
`sorted` returns a `Buffer` but not a `Seq`.